pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -3267,6 +3267,44 @@ public void getLastMessageId(
         }
     }
 
+    @PUT
```

Comment:
And is it better to change to @POST?
As I understand, the @PUT method usually used to create some resources.